src/app/shared/services/restaurant.service.ts

```diff
@@ -1,55 +1,22 @@
-/* eslint-disable @typescript-eslint/no-empty-function */
 import { Injectable } from '@angular/core';
 import { ItemCardInterface } from '../interfaces/restaurant-card.interface';
+import { HttpClient } from '@angular/common/http';
+import { Observable} from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 
 export class RestaurantService {
-  private restaurants: ItemCardInterface[] = [
-    {
-      name: 'Italiano1',
-      rating: 4.5,
-      reviews: 749,
-      imageUrl: 'https://i.pinimg.com/236x/6e/b6/91/6eb6911ae012db6c355671b231003d6e.jpg',
-      tags: ['COFFEE', 'DRINKS'],
-      delivery: {
-        isFree: true,
-        icon: 'motorcycle',
-        text: 'free delivery',
-      },
-      time: {
-        icon: 'access_time',
-        text: '10-15 mins',
-      },
-    },
-    {
-      name: 'Italiano2',
-      rating: 4.5,
-      reviews: 749,
-      imageUrl: 'https://i.pinimg.com/736x/a6/e1/8f/a6e18f7038d0e901d70e872f53ebf818.jpg',
-      tags: ['BURGER', 'CHICKEN'],
-      delivery: {
-        isFree: true,
-        icon: 'motorcycle',
-        text: 'free delivery',
-      },
-      time: {
-        icon: 'access_time',
-        text: '10-15 mins',
-      },
-    },
-  ];
-
+  private apiUrl = 'http://localhost:3000/restaurants'; 
```

Comment:
the url for the BE localhost should be stored inside your environment file: `src -> environments -> environment.ts` for the future production you will add in future `environment.prod.ts` with URLs for production.
Inside that file you write like this: 
```ts
export const environment = {
    apiUrl: 'http://localhost:3000',
}
``` 
So, as a result, inside your service, you will get 
```suggestion
  private apiUrl = '${environment.apiUrl}/restaurants'; 
```